src/services/userService.js

```diff
@@ -39,11 +39,32 @@ async function login(username, password) {
     }
 }
 
-async function getUserByUsername(username) {
-    const result = await userDAO.queryByUsername(username);
+const updateRole = async (id, role) => {
```

Comment:
No indentation?

Reply:
What should be indented here?

Reply:
It looks like the const updateRole is not indented to be inside getUserByUsername

Reply:
`async function getUserByUsername(username) {
    const result = await userDAO.queryByUsername(username);
const updateRole = async (id, role) => {`

It looks like this to me

Reply:
`updateRole` isn't inside `getUserByUsername` since it is it's own function, I removed `getUserByUsername` from the service layer since it's currently not used.

Reply:
Oh I see now. I was reading too fast and didn't notice it was deleted.